rules/windows/process_creation/proc_creation_win_msdt_diagcab.yml

```diff
@@ -15,8 +16,8 @@ detection:
     - OriginalFileName: 'msdt.exe'
   selection_cmd:
     CommandLine|contains|all:
-      - '/cab'
-      - '.diagcab'
```

Comment:
With DogWalk when clic on diagcab you have the `-cab` in CommandLine ?

Reply:
I made it more generic than DogWalk to take this case into consideration (https://twitter.com/nas_bench/status/1537896324837781506). But for the case of DogWalk you're correct it uses the "/"

Reply:
Even if you have a user who clic on a bad diagcab file found on the web ?
Never underestimate the ability of users to click on anything 🤣 
It is more phishing way  like "Fix all your slow internet problems with IamAgoodFile.diagcab"

Reply:
True users will click anything 😂 But to not create another rule for the command line case what do you suggest, should I restore the ".diagcab" extension and add the flag "-" or just create a new rule for the use case I linked above?

Reply:
I prefer to keep the original rule and add a new one with the new ref (https://twitter.com/nas_bench/status/1537896324837781506)  like proc_creation_win_msdt_susp_cab_option.yml .
So can play with filter and level .